pkg/server/status.go

```diff
@@ -41,10 +41,6 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		Type:   runtime.NetworkReady,
 		Status: true,
 	}
-	// Load the latest cni configuration to be in sync with the latest network configuration
-	if err := c.netPlugin.Load(c.cniLoadOptions()...); err != nil {
-		log.G(ctx).WithError(err).Errorf("Failed to load cni configuration")
```

Comment:
suggest we get the status of the last async reload request and if there was an error, output it's failure here as well..

Reply:
further since it is async the time since the last successful reload and/or time till the next reload may actually be interesting from a status information perspective. For example, someone sets the timer to a week.